### PR TITLE
FIX: Alert on android should use long strings as mesage

### PR DESCRIPTION
**Pull request by marcosrdz**
## Summary by CodeRabbit

- **New Features**
	- Enhanced alert functionality by preventing duplicate alerts with the same content.
	- Improved alert display logic for different platforms, ensuring better user experience.
	- Redesigned alert configuration to use structured objects for buttons and options.

- **Bug Fixes**
	- Adjusted handling of alert titles for Android to prevent empty titles.
	- Refined error handling and alert presentation for network errors.

- **Documentation**
	- Updated function signature to reflect new parameters for better clarity.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: components/Alert.ts [hunk 1]
```diff
@@ -23,13 +24,21 @@ const presentAlert = ({
...
       break;
     default:
-      RNAlert.alert(title ?? message, title && message ? message : undefined);
+      if (Platform.OS === 'android') {
+        // Android-specific behavior: If no title, use an empty string as the title
+        RNAlert.alert(title || '', message);
+      } else {
+        // For other platforms, use default logic
+        RNAlert.alert(title ?? message, title && message ? message : undefined);
+      }
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply components/Alert.ts]: No. need different logic per platform as the UI is different

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by Overtorment — APPROVED** [review]

**Pull request merged**